howso/client/base.py

```diff
@@ -3894,11 +3887,9 @@ def react_aggregate(  # noqa: C901
             non-robust type.
         prediction_stats_action_feature : str, optional
             When calculating residuals and prediction stats, uses this target features's
-            hyperparameters. The trainee must have been analyzed with this feature as the
-            action feature first. If both ``prediction_stats_action_feature`` and
-            ``action_feature`` are not provided, by default residuals and prediction
-            stats uses targetless hyperparameters. If "action_feature" is provided,
-            and this value is not provided, will default to ``action_feature``.
+            hyperparameters. The trainee should have been analyzed with this feature as the
+            action feature first. If ``prediction_stats_action_feature`` is not provided,
```

Comment:
Since this is the `prediction_stats_action_feature` parameter, you can just say `If not provided, ...` instead of naming it again.